sys/net/link_layer/ble_ll/ble_ll.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (C) 2015 James Hollister
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     ble_ll
+ * @file
+ * @brief       Implementation of the bluetooth low energy link layer
+ *
+ * @author      James Hollister <jhollisterjr@gmail.com>
+ * @}
+ */
+
+#include <errno.h>
+
+#include "kernel.h"
+#include "msg.h"
+#include "thread.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+#include "net/ble_ll.h"
+
+#define ENABLE_DEBUG    (1)
+#define ENABLE_BLE_PKT_DEBUG	(0)
```

Comment:
We had several discussions about logging (e.g.  #3506). Could you adapt this and prevent introducing a separate logging mechanism?